sdk/dotnet/Pulumi.Automation/LocalWorkspace.cs

```diff
@@ -36,7 +36,7 @@ public sealed class LocalWorkspace : Workspace
         private readonly LocalSerializer _serializer = new LocalSerializer();
         private readonly bool _ownsWorkingDir;
         private readonly Task _readyTask;
-        private static readonly SemVersion _minimumVersion = SemVersion.Parse("3.1.0");
+        private static readonly SemVersion _minimumVersion = new SemVersion(3, 1, 0);
```

Comment:
All these `SemVersion`-related changes were made because `SemVersion.Parse` was made obsolete. The equivalent is passing `SemVersionStyles.Any`, which is the most permissive format.